lit-cli/client.py

```diff
@@ -0,0 +1,149 @@
+    shutil.copytree('lit-base',package_name)
+    np.save(package_name+'/.lit-info.npy',{'package_name':package_name,'role':'','model_bucket':'serverless-server-models'})
+
+
+@click.command()
+@click.argument('package_name')
+@click.option('--requirement', '-r', is_flag=True, help='Install from the given requirements file')
+def install(package_name, requirement):
+    """Install dependencies specified by PACKAGE_NAME"""
+    do_install(package_name, requirement)
+
+
+@click.command()
+@click.option('--function-name', default='', help='Function name on Lambda Management Console')
+@click.option('--region', default='us-east-2', help='AWS region')
+@click.option('--role', default='', help='AWS IAM role')
+@click.option('--memory-size', default='1536', help='Lambda memory size')
+@click.option('--timeout', default='60', help='Lambda TTL')
+@click.option('--publish/--no-publish', default=False, help='Publish a new version or not')
+def deploy(function_name, region, role, memory_size, timeout, publish):
+    """Deploy package to AWS Lambda"""
+    info = np.load('.lit-info.npy').item()
+    info['function_name'] = function_name
+    if info['function_name'] == '':
+        info['function_name'] = info['package_name']
+    info['region'] = region
+    if role == '' and info['role'] == '':
+        raise ValueError('Unspecified IAM role')
+    if role != '':
+        info['role'] = role
+    info['memory_size'] = memory_size
+    info['timeout'] = timeout
+    subprocess.call('zip -9r lambda_function.zip  * -x *.params *symbol.json *.onnx', shell=True)
+    cmd = 'aws lambda create-function'
+    cmd += ' --function-name ' + info['function_name']
+    cmd += ' --zip-file fileb://lambda_function.zip'
+    cmd += ' --runtime python2.7'
+    cmd += ' --region ' + info['region']
+    cmd += ' --role ' + info['role']
+    cmd += ' --handler lambda_function.lambda_handler'
+    cmd += ' --memory-size ' + info['memory_size']
+    cmd += ' --timeout ' + info['timeout']
+    if publish:
+        cmd += ' --publish'
+    np.save('.lit-info.npy',info)
+    subprocess.call(cmd, shell=True)
+
+
+@click.command()
+@click.option('--memory-size', default=None, help='Lambda memory size')
+@click.option('--timeout', default=None, help='Lambda TTL')
+@click.option('--publish/--no-publish', default=False, help='Publish a new version or not')
+def update(memory_size, timeout, publish):
+    """Update package to AWS Lambda"""
+    info = np.load('.lit-info.npy').item()
+    info['memory_size'] = memory_size or info['memory_size']
+    info['timeout'] = timeout or info['timeout']
+    subprocess.call('zip -9r lambda_function.zip  *', shell=True)
+    cmd = 'aws lambda update-function-configuration'
+    cmd += ' --function-name ' + info['function_name']
+    cmd += ' --region ' + info['region']
+    cmd += ' --memory-size ' + info['memory_size']
+    cmd += ' --timeout ' + info['timeout']
+    if publish:
+        cmd += ' --publish'
+    subprocess.call(cmd, shell=True)
```

Comment:
what about the errors generated from this command?

Reply:
Thanks! Error handling added